src/Item/SpacerItem/Item.js

```diff
@@ -3,9 +3,10 @@ import i18n from 'd2-i18n';
 import ItemHeader from '../ItemHeader';
 
 const style = {
-    margin: '10px',
-    fontSize: '15px',
+    margin: '21px 28px',
+    fontSize: '14px',
     lineHeight: '18px',
+    color: '#666',
```

Comment:
Are you sure this color cant be approximated by one of the colors in colors.js (which are defined by Joe's UI color scheme)?